templates/repo/issue/labels/edit_delete_label.tmpl

```diff
@@ -26,31 +26,42 @@
 		<form class="ui edit-label form ignore-dirty" action="{{$.Link}}/edit" method="post">
 			{{.CsrfTokenHtml}}
 			<input id="label-modal-id" name="id" type="hidden">
-			<div class="ui grid">
-				<div class="three wide column">
-					<div class="ui small input">
-						<input class="new-label-input emoji-input" name="title" placeholder="{{.locale.Tr "repo.issues.new_label_placeholder"}}" autofocus required maxlength="50">
-					</div>
+			<div class="required field">
+				<label for="name">{{.locale.Tr "repo.issues.label_title"}}</label>
+				<div class="ui small input">
+					<input class="new-label-input emoji-input" name="title" placeholder="{{.locale.Tr "repo.issues.new_label_placeholder"}}" autofocus required maxlength="50">
 				</div>
-				<div class="five wide column">
-					<div class="ui small fluid input">
-						<input class="new-label-desc-input" name="description" placeholder="{{.locale.Tr "repo.issues.new_label_desc_placeholder"}}" maxlength="200">
-					</div>
+			</div>
+			<div class="field">
+				<div class="ui checkbox">
+					<input class="new-label-exclusive" name="exclusive" type="checkbox">
+					<label>{{.locale.Tr "repo.issues.label_exclusive"}}</label>
```

Comment:
```suggestion
					<label for="exclusive">{{.locale.Tr "repo.issues.label_exclusive"}}</label>
```
I guess?

Reply:
No, the `for` will be added automatically.  You have reviewed that:

* https://github.com/go-gitea/gitea/pull/22599

Reply:
Aargh.
It always gets me what Fomantic will autogenerate, and what it won't generate…